src/PIL/GifImagePlugin.py

```diff
@@ -704,11 +704,15 @@ def _get_global_header(im, info):
         if im.info.get("version") == b"89a":
             version = b"89a"
 
+    background = 0
+    if "background" in info:
+        background = info["background"]
+        if isinstance(background, tuple):
```

Comment:
Since this is a kind of hack, please add some comments about why we are doing this.

Reply:
Okay, done